hadoop-ozone/ozonefs/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFSInputStream.java

```diff
@@ -152,4 +154,12 @@ public void testO3FSByteBufferRead() throws IOException {
       Assert.assertArrayEquals(value, buffer.array());
     }
   }
+
+  @Test
+  public void testHasCapability() throws IOException {
+    try (OzoneFSInputStream inputStream =
+             new OzoneFSInputStream(fs.open(filePath))) {
```

Comment:
There is a compile error in CI:

```
2019-12-23T18:38:08.9765142Z [ERROR] /github/workspace/hadoop-ozone/ozonefs/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFSInputStream.java:[161,14] constructor OzoneFSInputStream in class org.apache.hadoop.fs.ozone.OzoneFSInputStream cannot be applied to given types;
2019-12-23T18:38:08.9775744Z   required: java.io.InputStream,org.apache.hadoop.fs.FileSystem.Statistics
2019-12-23T18:38:08.9804410Z   found: org.apache.hadoop.fs.FSDataInputStream
2019-12-23T18:38:08.9816806Z   reason: actual and formal argument lists differ in length
```

To fix it, please merge latest master (which introduced the new `Statistics` parameter in `OzoneFSInputStream` constructor) into the source branch.

Reply:
Thanks @adoroszlai for this info.
Rebase and fixed.